dlk/python/dlk/templates/src/func/conv2d.cpp

```diff
@@ -54,7 +54,7 @@ void conv3x3_kn2row(T input[],
   assert(p.input_height > 0);
   assert(p.input_width > 0);
 
-  U *buf = new U[MAX_SIZE_KN2ROW_BUFFER_PER_LAYER]();
+  static T_FLOAT buf[MAX_SIZE_KN2ROW_BUFFER_PER_LAYER];
```

Comment:
could we make it be
```cpp
static U buf[MAX_SIZE_KN2ROW_BUFFER_PER_LAYER]
```
?

as it was supposed to be a template function...

or making everything to be `T_FLOAT` instead of `U` for consistency is also a way to go, as we only use the functions in this file, thus they have not to be templated anyway.

Reply:
I see. I will try it too. Wait a moment please.